packages/styled-components/src/models/StyledComponent.js

```diff
@@ -30,6 +30,7 @@ import { useStyleSheet, useStylis } from './StyleSheetManager';
 import { ThemeContext } from './ThemeProvider';
 
 const identifiers = {};
+const isEnvDevelopment = process.env.NODE_ENV !== 'production';
```

Comment:
isEnvDevelopment is misleading, pick one:

```suggestion
const isNotProduction = process.env.NODE_ENV !== 'production';
```

```suggestion
const isDevelopment = process.env.NODE_ENV === 'development';
```

Reply:
in case if this app has more than 2 environments, anyway I've fixed it